htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/TextInput.java

```diff
@@ -37,6 +37,15 @@ public void sendKeys(CharSequence... keys) {
     }
 
     /**
+     * Sets this text input's value.
```

Comment:
Perhaps be explicit about using the `getClearCharSequence()` and not using `clear()`?